### DISCUSSION TITLE
Only show inner texture of glTF file

### DISCUSSION THREAD

**Discussion by sepehr500** [visgl/deck.gl#6890] [Q&A]
Hello. I am using SceneGraph glTF model of the inside of a house. It work great, but the textures are projected on the both sides of the surfaces, making it so only first person views make sense, and blocking view to the inside unless you zoom in past the wall. Is there a way to make textures only be visible on the side they are captured on? This would allow for a nice dollhouse view of the home. I see it done on this website https://gltf-viewer.donmccurdy.com/ but how do this in deck.gl?

**Comment by Pessimistress** [comment]
@belom88

**Comment by sepehr500** [comment]
Hi. Following up on this. Any help would be appreciated.
